css/index.css

```diff
@@ -104,14 +156,51 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
+    background: teal;
+    border: 1px solid red;
+}
+
+/* This is where I styled the colors of the boxes */
+
+#box1 {
+    background: teal;
+}
+ #box2{
+    background: gold;   
+}
+ #box3 {
+    background: cadetblue;
+}
+ #box4{
+    background: coral;
+}
+ #box5{
+    background: crimson;   
+}
+ #box6{
+    background: forestgreen;   
+}
+ #box7{
+    background: darkorchid;   
+}
+ #box8{
+    background: hotpink;    
+}
+ #box9{
+    background: indigo;  
+}
+ #box10{
+    background: dodgerblue;   
```

Comment:
Good job on implementing dry programming here by only applying the background color to each individual box and then applying the same css to all of the boxes with a separate classname. 👏💯